playbooks/README.md

```diff
@@ -97,6 +96,10 @@ Currently we don't have a playbook which installs KubeVirt on a Kubernetes clust
 
 ### OpenShift cluster
 
+The playbook expects you have the client installed and a config file in
```

Comment:
@rthallisey what client? and a link on installation step docs will be appreciated.